source/blog/2018-07-19-ovn-dpdk.html.md

```diff
@@ -41,15 +41,15 @@ An example playbook:
     - ovirt-ansible-dpdk-setup
 ```
 The role is controlled by 3 boolean variables (all set to `true` by default) and a dictionary of devices and their drivers:
-* `configure_kernel` -- determines whether the kernel should be configured for DPDK usage (hugepages, CPU partitioning)
+* `configure_kernel` -- determines whether the kernel should be configured for DPDK usage (hugepages, CPU partitioning). WARNING: When set to `true` it is very likely to trigger reboot of the host, unless all required configuration is already pre-set.
 * `bind_drivers` -- determines whether the role should bind devices to their specified drivers.
 * `set_ovs_dpdk` -- determines whether the role should initialize and configure OVS for DPDK usage.
-* `pci_drivers` -- a dictionary of the PCI addresses of network interface cards as keys and drivers as values. Empty strings represent the kernel default. non-DPDK compatible drivers won't be configured for OVS-DPDK usage.
+* `pci_drivers` -- a dictionary of the PCI addresses of network interface cards as keys and drivers as values. Empty strings represent kernel's default. non-DPDK compatible drivers may be set; NICs not using DPDK compatible drivers won't be configured for OVS-DPDK usage and won't have their CPU isolated in multiple NUMA environments.
```

Comment:
still not clear if specifying a bad pci device, we end up with an error code

Reply:
As in non existent device? yes, we'll explode, but I think that's a given? What else should be expected from specifying non existing devices? A fail silently scenario should be explicitly documented, not the other way around imo

Reply:
oh, now I see that I was misguided by the double (or triple, or quadruple) negation. Consider turning

non-DPDK compatible drivers may be set; NICs not using DPDK compatible drivers won't be configured for OVS-DPDK usage and won't have their CPU isolated in multiple NUMA environments.

into

Non-DPDK compatible drivers may be set; However, NICs with such drivers will not be configured for OVS-DPDK usage and will not have their CPU isolated in multiple NUMA environments.

Reply:
I'm not sure that this is precise: "Empty strings represent kernel's default"

"The empty string represents the default Linux driver for the specified device"